config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presets.yaml

```diff
@@ -12,3 +12,21 @@ presets:
       secretKeyRef:
         name: azure-capz-sa-cred
         key: serviceAccountSigningKey
+- labels:
+    preset-azure-cred-wi: "true"
+  env:
+    - name: AZURE_WORKLOAD_ID
+      value: "cabf5f22-ec7e-4e84-9e35-c02e57ca555d" # not a secret
```

Comment:
Let's add sub and tenant id here too.  that way we can remove key that adds the Secret that we currently have.  This way we know if we are missing anything

Reply:
Added via 379bea431d845e5de172a8b8dec9f255f248ae79